src/app/clients/ClientsPage.tsx

```diff
@@ -21,6 +21,8 @@ import { CenterComponent } from "@/components/Form/formStyling";
 import ActionBar from "@/app/clients/ActionBar";
 import AddParcelsButton from "@/app/clients/AddParcelsButton";
 
+// TODO Change Button to LinkButton
```

Comment:
wait this is no longer on main - did you rebase recently? please remove now :) [sorry]

Reply:
It's all good, I noticed that when I rebased back onto `main`